_posts/2022-12-16-aave-rebasing.md

```diff
@@ -0,0 +1,313 @@
+
+In this post, we're going to explain how Aave calculates interest on deposits,
+and how their approach made it possible for us to implement a flex voting
+aToken **[TODO link to our contract in github]**.
+
+### Flexible Voting
+
+[Flexible voting](https://www.scopelift.co/blog/introducing-flexible-voting) (or _flex voting_ for short) is a novel extension to Compound-style governance systems.
+It allows delegates to split their voting weight across For/Against/Abstain options, rather than having to put all of it behind one.
+
+Enabling flex voting for Aave would unlock a large amount of value for users.
+
+To see why, imagine you hold UNI.
+UNI gets its intrinsic value from being the voting token for [Uniswap governance proposals](https://app.uniswap.org/#/vote).
+Anyone who holds UNI can delegate and vote on these proposals and help set the course of one of the biggest projects in crypto.
+
+So the main value of UNI comes from its use in voting.
+But it would be really nice if you could also deposit your UNI to earn yield at the same time.
+
+The trouble is that if you were to deposit UNI into a DeFi protocol (like Aave) you would lose your voting power.
+This is because the address that holds the tokens controls their voting weight.
+And if you deposit your UNI, the receiving contract -- not you -- would hold the tokens.
+
+So governance token holders are largely forced to choose: either participate in governance or earn yield.
+They cannot currently have both.
+
+If flex voting were added to Aave, however, they could.
+
+In such a world, Aave depositors would express their desired voting preference to
+Aave, which would then cast its vote in the appropriate ratios to
+the governance contract.
+
+So, for example, if 35% of UNI depositors expressed a "No" voting preference on a
+given proposal, 60% expressed "Yes", and 5% "Abstain", then Aave
+could cast its vote to the UNI governance system with 35% of its weight as
+Against, 60% of its weight as For, and 5% as Abstain.
+
+Aave users would effectively get to vote without having a UNI balance!
+
+While flex voting makes this dream scenario possible, it's not without technical hurdles.
+The next section will get into what these are and how we solved them.
+
+### aToken Rebasing ELI5
+
+When users deposit tokens into Aave, they receive [ERC20](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/) tokens in return.
+Aave's ERC20's are called [aTokens](https://github.com/aave/aave-v3-core/blob/f3e037b3638e3b7c98f0c09c56c5efde54f7c5d2/contracts/protocol/tokenization/AToken.sol).
+If you deposited [DAI](https://polygonscan.com/address/0x8f3cf7ad23cd3cadbd9735aff958023239c6a063) on Polygon, you would receive [aPolDAI](https://polygonscan.com/address/0x82E64f49Ed5EC1bC6e43DAD4FC8Af9bb3A2312EE) back.
+[Here](https://docs.aave.com/developers/deployed-contracts/v3-mainnet/polygon#tokens) is a list of other commonly-issued aTokens.
```

Comment:
Related nit: link to optimism tokens / use them in examples instead, since our repo forks from optimism
```suggestion
[Here](https://docs.aave.com/developers/deployed-contracts/v3-mainnet/polygon#tokens) is a list of other commonly-issued aTokens on Polygon.
```